features/src/rapids-build-utils/opt/rapids-build-utils/bin/tmpl/python.build.wheel.tmpl.sh

```diff
@@ -68,6 +68,7 @@ build_${PY_LIB}_python_wheel() {
         $(rapids-select-pip-wheel-args "$@")
     )";
 
+    # TODO: Should this be handling the _skbuild directory from scikit-build differently?
```

Comment:
No, because the path to `_skbuild` is hard-coded in scikit-build. This check here is telling `scikit-build-core` to build into `build/pip/cuda12.2/release`.